cypress/support/e2e.js

```diff
@@ -17,6 +17,7 @@
  */
 
 // Importing generic module
```

Comment:
This comment should either go away or stay above "import * as genericModule"

Reply:
Done